### PR TITLE
Adding highlighting issue #26

### PR DESCRIPTION
**Pull request by abdrabrcom**
As per Issue #26 syntax Highlighting function has been implemented and unit test provided

### CODE REVIEW

File: com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/Highlights.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+package com.ca.lsp.cobol.service.delegates;
```
Comment (shciubrcom) [inline com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/Highlights.java]: Is it still ca package?

Comment (temanbrcom) [reply com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/Highlights.java]: We didn't apply the renaming yet.

File: com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/HighlightsTest.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  @Before
+  public void initializeService() {
+    TestLanguageClient client = new TestLanguageClient();
+    service = createServer(client);
+    runTextValidation(service, TEXT);
+    waitForDiagnostics(client);
+  }
+
+  /*
+   * Testing if Range from Highlights function is providing proper ranges against EXPECTED_RANGES
```
Comment (temanbrcom) [inline com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/HighlightsTest.java]: This comment should be a part of the test javadoc

Comment (abdrabrcom) [reply com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/HighlightsTest.java]: Correction applied

**Comment by tomascechatbroadcomcom** [comment]
don't forget to merge latest development branch.

**Pull request merged**